tutorial/part3/data_exploitability_pangeo.ipynb

```diff
@@ -3,7 +3,10 @@
   {
```

Comment:
<ul><li>Justus Magin, UMR-LOPS CNRS(France), @keewis</li><li>Tina Odaka, UMR-LOPS Ifremer (France),&nbsp;<a href="https://github.com/tinaok" target="_blank">@tinaok</a></li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-data/pangeo-openeo-BiDS-2023/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
@tinaok this is fixed in the latest push in this PR

Reply:
We should update your affiliation in the other notebooks

Reply:
@acocac could you please fix my name? It's Michele Claus, not Clous

Reply:
@clausmichele fixed!

Reply:
my new affiliation is:  Provare LTD